### ISSUE TITLE
Support firebase.json that isn't in the root of the repo

### ISSUE THREAD

**Issue by jhuleatt** [FirebaseExtended/action-hosting-deploy#5]
For cases where the `firebase.json` file isn't at the root of the repo, we should allow configuration of the path to point to a specific spot.

For example:

```yml
- uses: FirebaseExtended/action-hosting-deploy@v0
  with:
    repoToken: "${{ secrets.GITHUB_TOKEN }}"
    firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT }}"
    projectId: my-project
    path: "./website" # Action will cd into this path before running
```

**Comment by jhuleatt** [comment]
or maybe this can be solved in the workflow file with the `entrypoint` key?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswithentrypoint

**Comment by jhuleatt** [comment]
no:

> The entrypoint keyword is meant to use with Docker container actions, but you can also use it with JavaScript actions that don't define any inputs.)

**Comment by jhuleatt** [comment]
Fixed by #16
